Volume/Views/Onboarding/PublicationDetail/PublicationDetail.swift

```diff
@@ -139,17 +149,17 @@ struct PublicationDetail: View {
                     }
                 }
             }
-            .edgesIgnoringSafeArea(.top)
-            .navigationBarHidden(true)
             .disabled(isLoading)
         }
+        .edgesIgnoringSafeArea(.top)
+        .navigationBarHidden(true)
         .background(Color.white)
```

Comment:
`.white`?

Reply:
This won't compile, because `background(_ label:)` takes in `some View`.